src/IntelliTect.Coalesce.CodeGeneration/Generation/Generators/Generator.cs

```diff
@@ -51,7 +52,8 @@ public virtual void Configure(JObject obj)
             foreach (var configProp in properties)
             {
                 var propType = configProp.Type.TypeInfo;
-                configProp.PropertyInfo.SetValue(this, obj.ToObject(propType));
+                JToken value = obj.GetValue(configProp.Name, StringComparison.OrdinalIgnoreCase);
+                configProp.PropertyInfo.SetValue(this, value.ToObject(propType));
```

Comment:
Is this robust against nulls when the token that is searched for in `JToken value = obj.GetValue...` isn't found? (i.e. the config property isn't defined in coalesce.json).

Reply:
Will test